diff -aurN pmount-0.9.7.orig/src/Makefile.am pmount-0.9.7/src/Makefile.am
--- pmount-0.9.7.orig/src/Makefile.am	2005-12-16 11:34:37.000000000 +0100
+++ pmount-0.9.7/src/Makefile.am	2006-02-27 16:23:03.876543872 +0100
@@ -30,8 +30,8 @@
 
 install-data-hook:
 	test -z $(INSTALL_DIR) || mkdir -p -- $(INSTALL_DIR)
-	$(INSTALL_DATA) -o root -g root -m 4755 -D $(INSTALL_SRC)/pmount $(INSTALL_DIR)/pmount
-	$(INSTALL_DATA) -o root -g root -m 4755 -D $(INSTALL_SRC)/pumount $(INSTALL_DIR)/pumount
+	$(INSTALL_DATA) -D $(INSTALL_SRC)/pmount $(INSTALL_DIR)/pmount
+	$(INSTALL_DATA) -D $(INSTALL_SRC)/pumount $(INSTALL_DIR)/pumount
 	
 uninstall-hook:
 	rm -f $(INSTALL_DIR)/pmount
